src/utils/remove-extension.js

```diff
@@ -0,0 +1,7 @@
+// Removes file extension from file name if asset is an image
```

Comment:
This comment doesn't quite apply anymore (I think I did this!)

Reply:
Oops, that was me, just forgot to remove it